chapter-03/implementing-server-side-kotlin-development/src/test/kotlin/com/example/implementingserversidekotlindevelopment/domain/CreatedArticleTest.kt

```diff
@@ -18,7 +18,7 @@ class CreatedArticleTest {
     fun articleEqualTest(): Stream<DynamicNode> {
         return Stream.of(
             TestCase(
-                "Slug が一致する場合、他のプロパティが異なっていても、true を戻す",
+                "ArticleId が一致する場合、他のプロパティが異なっていても、true を戻す",
```

Comment:
[nits]

```suggestion
                "Slug が一致する場合、他のプロパティが異なっていても、true を戻す",
```

typo があるので修正。